code-studio/scripts/build-js.js

```diff
@@ -63,6 +63,15 @@ Promise.all([
       'embedVideo.js'
     ],
     commonFile: 'embedVideo'
+  })),
+
+  // code-studio-globals.js is its own bundle so that we can include some globals
+  // without needing to load the rest of code-studio.
+  build_commands.bundle(_.extend({}, defaultOptions, {
+    filenames: [
+      'code-studio-globals.js'
+    ],
+    commonFile: 'code-studio-globals'
```

Comment:
Sure - let's call it something like `only-react` for now (since it's not like we've removed React from the main code-studio bundle, or are including these 'globals' in most code-studio pages).  If you want to be more generic, let's map it to the only place this is currently used: 'embed_blocks'.